mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -51,6 +51,17 @@ trait Completions { this: MetalsGlobal =>
   class NamedArgMember(sym: Symbol)
       extends ScopeMember(sym, NoType, true, EmptyTree)
 
+  class DependecyMember(val dependency: String, val edit: l.TextEdit)
+      extends ScopeMember(
+        completionsSymbol(dependency),
+        NoType,
+        true,
+        EmptyTree
+      ) {
+    val label: String = dependency.stripPrefix(":")
+    def isVersion: Boolean = label.headOption.exists(_.isDigit)
```

Comment:
This might not work if we have a dependency name starting with a digit

Reply:
Maybe replace strip anything after `-` and check if the rest is dots and numbers?